stemcell_builder/stages/base_ubuntu_packages/apply.sh

```diff
@@ -14,11 +14,25 @@ libaio1 gdb libcap2-bin libcap2-dev libbz2-dev \
 cmake uuid-dev libgcrypt-dev ca-certificates \
 scsitools mg htop module-assistant debhelper runit parted \
 cloud-guest-utils anacron software-properties-common"
+
+if [ `uname -m` == "ppc64le" ]; then
+  debs="$debs \
+libreadline-dev libtool texinfo ppc64-diag libffi-dev \
+libruby bundler libgmp-dev libgmp3-dev libmpfr-dev libmpc-dev"
+fi
+
 pkg_mgr install $debs
 
 # we need newer rsyslog; this comes from the upstream project's own repo
 run_in_chroot $chroot "add-apt-repository ppa:adiscon/v8-stable"
-pkg_mgr install "rsyslog rsyslog-relp rsyslog-mmjsonparse rsyslog-gnutls"
+# needed to remove rsyslog-mmjsonparse on ppc64le
+# because of this issue https://gist.github.com/allomov-altoros/cd579aa76f3049bee9c7
+if [ `uname -m` == "ppc64le" ]; then
+  pkg_mgr install "rsyslog rsyslog-relp rsyslog-gnutls"
```

Comment:
could always run `pkg_mgr install "rsyslog rsyslog-relp rsyslog-gnutls"` and then install conditionally rsyslog-mmjsonparse